javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,44 @@
+let userName = [];
+let myTodo = [];
+function getReply(command) {
+  if (command.includes("my name is")) {
+    let commandStr = command.split(" ");
+    let activeName = commandStr[commandStr.length - 1];
+
+    if (userName.includes(activeName)) {
+      console.log(" Yes I know");
+      return;
+    }
+
+    userName.push(activeName);
+    console.log(`nice to meet you ${activeName}`);
+    return;
+  }
+
+  if (command.includes("what is my name")) {
+    if (!userName) {
+      console.log("what is your name");
+      return;
+    }
+    console.log(`your name is ${activeName}`);
+    return;
+  }
+  if (command.includes("Add fishing to my todo")) {
```

Comment:
This should be dynamic. 
You could do something like:
```js
if( 
  command.includes("Add") 
  && command.includes("to my todo")
)
```

Reply:
I have completed this exercise.